generated/app/ApicollectiveApibuilderSpecV0Client.scala

```diff
@@ -201,7 +201,8 @@ package io.apibuilder.spec.v0.models {
     `type`: String,
     headers: _root_.scala.Option[Seq[io.apibuilder.spec.v0.models.Header]] = None,
     description: _root_.scala.Option[String] = None,
-    deprecation: _root_.scala.Option[io.apibuilder.spec.v0.models.Deprecation] = None
+    deprecation: _root_.scala.Option[io.apibuilder.spec.v0.models.Deprecation] = None,
+    attributes: Seq[io.apibuilder.spec.v0.models.Attribute] = Nil
```

Comment:
Doesn't look like this file got regenerated after the last model change. This should be Option[Seq[Attribute]], right?